glrd/manage.py

```diff
@@ -623,6 +678,9 @@ def create_single_release(release_type, args, existing_releases):
         release['git']['commit_short'] = commit_short
         release['github'] = {}
         release['github']['release'] = f"https://github.com/gardenlinux/gardenlinux/releases/tag/{major}.{minor}"
+        release['flavors'] = flavors
+        release['attributes'] = {}
+        release['attributes']['source_repo'] = True 
```

Comment:
Not part of this PR - Too many lines and too many conditional code. Clear signs for refactoring.

Reply:
https://github.com/gardenlinux/glrd/issues/4